### PR TITLE
bug 1683753 added note for ansible playbooks

### PR DESCRIPTION
**Pull request by sheriff-rh**
Link to bug:
https://bugzilla.redhat.com/show_bug.cgi?id=1683753

@gpei  PTAL - also, please determine which releases this needs to be cherry-picked to. 

@openshift/team-documentation - peer review please :)

### CODE REVIEW

File: getting_started/install_openshift.adoc [hunk 1]
```diff
@@ -162,6 +162,12 @@ xref:../install/example_inventories.adoc#install-config-example-inventories[Exam
 Inventory Files] and select the example that most closely matches your desired cluster
 configuration.
 +
+
+[IMPORTANT]
+====
+Running Ansible playbooks with `--tags` or `--check` is not supported by Red Hat.
```
Comment (kalexand-rh) [inline getting_started/install_openshift.adoc]: s/with \`--tags\` or \`--check\`/with the \`--tags\` or \`--check\` options

**Review by kalexand-rh — COMMENTED** [review]
It's better to include nouns to describe the options/variables/files/constructs than just the constructs themselves.

File: getting_started/install_openshift.adoc (hunk 1 above)
Comment (sheriff-rh) [reply getting_started/install_openshift.adoc]: Agreed! Fixed and squashed. Thanks!

**Comment by sheriff-rh** [comment]
QA have approved this request and I will cherry-pick it to all 3.0+ releases. Merging.

**Pull request merged**